source/fundamentals/collations.txt

```diff
@@ -0,0 +1,280 @@
+"``fr_CA``" locale. The collation applies to all :manual:`operations that
+support collation </reference/collation/#operations-that-support-collation>`
+performed on that collection.
+
+.. literalinclude:: /code-snippets/collation/collection-collation.js
+   :language: javascript
+   :dedent: 4
+
+Assign a Collation to an Index
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In the following example, we create a new index on the ``title`` field of
+the ``movies`` collection in the ``sample_mflix`` database with a collation
+set to the "``en_US``" locale.
+
+.. literalinclude:: /code-snippets/collation/index-collation.js
+   :language: javascript
+   :dedent: 4
+
+The following query uses the index we created:
+
+.. literalinclude:: /code-snippets/collation/query-index-collation.js
+   :language: javascript
+   :start-after: start query index collation
+   :end-before: end query index collation
+   :dedent: 4
+
+The following queries **do not** use the index that we created.  The first
+query does not include a collation and the second contains a different
+strength value than the collation on the index.
+
+.. literalinclude:: /code-snippets/collation/query-index-collation.js
+   :language: javascript
+   :start-after: start query without index collation
+   :end-before: end query without index collation
+   :dedent: 4
+
+Collation Query Examples
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Operations that read, update, and delete documents from a collection can use
+collations. This section includes examples of a selection of these. See the
+MongoDB manual for a full list of :manual:`operations that support collation
+</reference/collation/#operations-that-support-collation>`.
+
+find() and sort() Example
+`````````````````````````
+
+The following example calls both ``find()`` and ``sort()`` on a collection
+called ``contacts`` in a database called ``myproject`` using a German collation
+with the ``locale`` parameter set to ``de``.
+
+.. literalinclude:: /code-snippets/collation/find-sort-collation.js
+   :language: javascript
+   :dedent: 4
+
+findOneAndUpdate() Example
+``````````````````````````
+
+The following example calls the ``findOneAndUpdate()`` operation on a
+collection called ``names`` in which contains the following documents:
+
+::
+
+  { "_id" : 1, "first_name" : "Hans" }
+  { "_id" : 2, "first_name" : "Gunter" }
+  { "_id" : 3, "first_name" : "Günter" }
+  { "_id" : 4, "first_name" : "Jürgen" }
+
+Consider the following ``findOneAndUpdate()`` operation on this collection
+which **does not** specify a collation:
+
+.. literalinclude:: /code-snippets/collation/findOneAndUpdate-collation.js
+   :language: javascript
+   :start-after: start findOneAndUpdate without collation
+   :end-before: end findOneAndUpdate without collation
+   :dedent: 4
+
+Since "Gunter" is the first result in the sorted results when using a binary
+collation, the operation does not update any documents. Consider the same
```

Comment:
I'd suggest explaining *why* the Gunter document doesn't update here.

Reply:
Great feedback, I think the newer version does not force the user to infer anything anymore.